### ISSUE TITLE
Does not work correctly with orthographic cameras

### ISSUE THREAD

**Issue by raveaf** [keijiro/KinoMotion#13]
Are orthographic cameras even supported?
[test_scene.zip](https://github.com/keijiro/KinoMotion/files/435195/test_scene.zip)

**Comment by keijiro** [comment]
I think it's working properly. Could you explain what the point is? Attaching a screenshot might help a lot :)

**Comment by keijiro** [comment]
FYI: I checked it with Unity 5.4.0p2

**Comment by catafest** [comment]
With Unity 5.3.5f1 (64-bit)not working for me , I will try with the new version of Unity.

**Comment by keijiro** [comment]
@catafest It requires Unity 5.4.0 or later.

**Comment by catafest** [comment]
the defaul scene has nothing to test it.
how is working ?
can you help me with this output over windows 10 **tree** command? The output file is strange ...

```
|   |   +---fc
|   |   +---fd
|   |   +---fe
|   |   \---ff
|   +---ScriptAssemblies
|   \---ShaderCache
|       +---0
|       +---2
|       +---3
|       +---4
|       +---5
|       +---6
|       +---7
|       +---9
|       +---a
|       +---b
|       +---c
|       +---d
|       \---f
+---ProjectSettings
\---Temp
    +---ProcessJobs
    \---__Backupscenes
```

**Comment by keijiro** [comment]
@catafest This issue ticket is dedicated to the problem that @raweaf met. Please create a new ticket for a new topic.

**Comment by catafest** [comment]
yes

**Comment by raveaf** [comment]
Weird, now it works and I can not reproduce it. Thanks anyway!

**Comment by raveaf** [comment]
I encountered the problem again a few times, but I always got it fixed by closing and reopening the game window. Maybe it is just a unity bug.

But I found a different problem when using an orthographic camera. The motion blur gets "occluded" when an opaque object is behind the object effected by the motion blur. It does not happen when the object behind is transparent, so maybe it has something to do with the depth buffer.

I also added a test project (now with the correct scene).

![test](https://cloud.githubusercontent.com/assets/4925469/17980501/eca0e0c6-6aff-11e6-8baf-bdb640c1f98c.png)
![test2](https://cloud.githubusercontent.com/assets/4925469/17980515/f65f8b94-6aff-11e6-8c3f-b1c5a81995c7.png)
[test_scene.zip](https://github.com/keijiro/KinoMotion/files/437745/test_scene.zip)

**Comment by keijiro** [comment]
Fixed! (b3f7a10) Thanks for reporting it!

FYI: It seems the skybox shader writes wrong motion vectors. This issue is not related to the Motion effect, so I'll report it to UTech later. At the moment, it's recommended to put a background quad or something to avoid artifacts.
